### ISSUE TITLE
Build CI example with Cartographer

### ISSUE THREAD

**Issue by petewall** [vmware-tanzu/asset-relocation-tool-for-kubernetes#144]
We currently have a CI example with Concourse. Mostly that's because I like that tool :). The example holds, and is still useful to show that this tool does work sufficiently in a CI setting. However, as a concrete example, it doesn't scale well for users that aren't already using Concourse.

Another integration that might be useful is [Cartographer](https://cartographer.sh/). This allows dynamic generation of pipelines as a customer wants to add new Helm Charts to the mix, and plugs into their existing supply chain assets for handling things like image scanning and multi-cluster deployment. It also dovetails nicely with the CD strategy that VMware Tanzu is investing in going forward.

The project would be to create a CluterImageTemplate for relok8s:
https://cartographer.sh/docs/v0.2.0/reference/template/#clusterimagetemplate

Basically, we'd be creating a Kubernetes resource that accepts as input an artifact and revision that points to the source Helm Chart, an image patterns file that is supplied as a parameter to the Workload, and executes relok8s. The resource would output as status an image reference to the relocated Helm Chart, which would gets passed off to the next stage in the supply chain (e.g. scanning or deployment)

**Comment by migmartri** [comment]
This is cool!

**Comment by petewall** [comment]
Not within scope. Closing for now
